src/commands/logs.ts

```diff
@@ -16,10 +16,18 @@ const padEnd = require('string.prototype.padend');
  * @param cmd.open {boolean} If true, the command will open the StackDriver logs website.
  * @param cmd.setup {boolean} If true, the command will help you setup logs.
  * @param cmd.watch {boolean} If true, the command will watch for logs and print them. Exit with ^C.
+ * @param cmd.simplified {boolean} If true, the command will add timestamps to the logs.
```

Comment:
If **false**, will add timestamps

Reply:
Changed wordage to "If true, the command will remove timestamps from the logs".